GC Manager/GCMMultiplayer.h

```diff
@@ -74,7 +74,12 @@
 /// @b Readonly. Indicates whether or not the multiplayer match has started (if one has been created).
 @property (nonatomic, assign, readonly) BOOL multiplayerMatchStarted;
 
+#if TARGET_OS_IPHONE
 @property (nonatomic, strong, readonly) UIViewController *matchPresentingController;
+#else
+@property (nonatomic, strong, readonly) NSViewController *matchPresentingController;
```

Comment:
While this resolves an issue in the header, it my also need to be fixed in the implementation.